jekyll/_cci2/configuration-reference.md

```diff
@@ -555,6 +555,31 @@ fingerprints | N | List | List of fingerprints corresponding to the keys to be a
 
 Note that CircleCI 2.0 builds are auto configured with `ssh-agent` with all keys auto-loaded, and is sufficient for most cases. `add_ssh_keys` may be needed to have greater control over which SSH keys to authenticate (e.g. avoid "Too many authentication failures" problem when having too many SSH keys).
 
+## **`workflows`**
+
+Used for orchestrating all jobs. Each workflow consists of the workflow name as a key and a map as a value. A name should be unique within the current `config.yml`. Jobs are run in parallel by default, so you must explicitly require any dependencies by their job name.
+
+Key | Required | Type | Description
+----|-----------|------|------------
+version | Y | String | Should currently be 2
+name | N | String | Title of the workflow to be shown in the CircleCI UI
+jobs | Y | List | A list of jobs to run with their dependencies
+requires | N | List | A list of jobs that must succeed for the job to start
+branches | N | Map | A map defining rules for execution on specific branches
+only | N | String | A branch name
```

Comment:
We'll need to split this into multiple tables or use some other method since the keys are only applicable at different parts of the YAML hierarchy.

The top level keys for workflows config are `version` and `jobs`

A job can have the keys `requires` and `filters`

`filters` can have the key `branches`

`branches` can have the keys `only` and `ignore` which either map to a single string naming a branch (or a regexp to match against branches, has to be enclosed with `/`s) or map to a list of such strings.
* Any branches that match `only` will run the job.
* Any branches that match `ignore` will not run the job.
* If neither `only` nor `ignore` are specified then all branches will run the job.
* If both `only` and `ignore` are specified the `only` overrides `ignore`."